stdlib/public/Distributed/DistributedActor.swift

```diff
@@ -255,7 +260,7 @@ public protocol DistributedActor: AnyActor, Identifiable, Hashable
   /// be introduced when not strictly required.  Visible side effects
   /// are therefore strongly discouraged within this property.
   @available(SwiftStdlib 5.9, *)
-  nonisolated var unownedExecutor: UnownedSerialExecutor { get }
```

Comment:
This requirement has never shipped and cannot have been used by anyone yet, so making sure we have the _right_ signature/requirement in the API/ABI for distributed actors by merging this now.